drakrun/drakrun/lib/postprocessing/capa_plugin/capa_processor.py

```diff
@@ -0,0 +1,373 @@
+import functools
+import itertools
+import logging
+import multiprocessing
+import shutil
+import zipfile
+from pathlib import Path
+from tempfile import TemporaryDirectory
+from typing import Any, Dict, Iterator, List, Optional, TextIO, Tuple, Union
+
+import capa.engine as ce
+import capa.features.address as ca
+import capa.render.result_document as crd
```

Comment:
I see there is a name collision and that's why you can't use `from x import y` but we can fix it by using an alias that is a bit more descriptive:
```suggestion
from capa.engine import MatchResults as EngineMatchResults
import capa.features.address as ca
from capa.render.result_document import MatchResults as ResultDocumentMatchResults
```

but it's just a note, I'm not strongly against these imports.